velox/connectors/hive/storage_adapters/s3fs/S3Util.h

```diff
@@ -57,8 +76,17 @@ inline std::string s3URI(const std::string& bucket, const std::string& key) {
 }
 
 inline std::string s3Path(const std::string_view& path) {
-  // Remove the prefix S3:// from the given path
-  return std::string(path.substr(kS3Scheme.length()));
+  // Remove one of the prefixes 's3://', 'oss://', 's3a://' if any from the
+  // given path.
+  // TODO: Each implementation should be registered as separate filesystem
```

Comment:
`// TODO: Each prefix should be implemented as its own filesystem.` ?

Reply:
Fixed.